integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/property/PropertyIntegrationNO_ENTITY_BODY_PropertyTest.java

```diff
@@ -37,7 +39,7 @@
 
 public class PropertyIntegrationNO_ENTITY_BODY_PropertyTest extends ESBIntegrationTest {
```

Comment:
This class name does not follows class naming standards. Shall we rename the class to a proper name?

Reply:
Will refactor this with another PR as this is already existing class.